usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -112,6 +112,22 @@ def last_closed_monthly_submission_dates(self):
     def last_closed_quarterly_submission_dates(self):
         return get_last_closed_submission_date(is_quarter=True)
 
+    @cached_property
+    def final_submissions_query_filters(self):
+        return filter_by_latest_submissions()
+
+    @cached_property
+    def all_closed_defc_submissions(self):
+        q = Q()
+        for final_for_fy in finals_for_all_fy():
+            if (final_for_fy[0] == 2020 and final_for_fy[2] >= 7) or final_for_fy[0] > 2021:
+                q |= (
+                    Q(submission__reporting_fiscal_year=final_for_fy[0])
+                    & Q(submission__quarter_format_flag=final_for_fy[1])
+                    & Q(submission__reporting_fiscal_period__lte=final_for_fy[2])
```

Comment:
Didn't we want _all_ closed periods up to and including the last closed. This will just get all the final_for_fy periods, but not periods prior to them in their FY (that were also closed)